benchmark/scripts/preflight.py

```
@@ -0,0 +1,229 @@
+    """Run all static preflight checks and print a compact success summary."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--env-file", type=Path, required=True)
+    args = parser.parse_args()
+    load_env_file(args.env_file)
+
+    litellm_base_url = os.environ.get("LITELLM_OPENAI_BASE_URL", "").strip()
+    opus_bridge_url = os.environ.get("CODEX_OPUS_RESPONSES_BASE_URL", "").strip()
+    if not litellm_base_url or not opus_bridge_url:
+        raise SystemExit(
+            "LITELLM_OPENAI_BASE_URL and CODEX_OPUS_RESPONSES_BASE_URL are required"
+        )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)